UPGRADE-5.4.md

```diff
@@ -38,6 +38,8 @@ Messenger
 SecurityBundle
 --------------
 
+ * Deprecate `security.authentication.basic_entry_point` and `security.authentication.retry_entry_point` services, the logic is moved into the
+   `HttpBasicAuthenticator` and `ContextListener` respectively
```

Comment:
s/ContextListener/ChannelListener in all markdown files :)

Reply:
Thanks :)